crates/ruff_python_formatter/src/statement/stmt_class_def.rs

```diff
@@ -136,3 +117,54 @@ impl FormatNodeRule<StmtClassDef> for FormatStmtClassDef {
         Ok(())
     }
 }
+
+pub(super) struct FormatDecorators<'a> {
+    pub(super) decorators: &'a [Decorator],
+    pub(super) leading_definition_comments: &'a [SourceComment],
+}
+
+impl Format<PyFormatContext<'_>> for FormatDecorators<'_> {
+    fn fmt(&self, f: &mut Formatter<PyFormatContext<'_>>) -> FormatResult<()> {
```

Comment:
For tomorrow: Should we instead add a `is_suppressed` method to `FormatNodeRule` and pass the `trailing_comments`. It would prevent the extra lookup of the trailing comments only to test whether the statement is suppressed or not. 

The downside of this is that we need to override the function for every single statement but that could be justified to safe on 2-3% formatting performance